README.md

```diff
@@ -2482,6 +2489,149 @@ Client.newHazelcastClient().then(function (client) {
+  - `RANDOM`: A random item is evicted.
+  
+- `evictionMaxSize`: Maximum number of entries kept in the memory before eviction kicks in.
+- `evictionSamplingCount`: Number of random entries that are evaluated to see if some of them are already expired. If there are expired entries, those are removed and there is no need for eviction.
+- `evictionSamplingPoolSize`: Size of the pool for eviction candidates. The pool is kept sorted according to eviction policy. The entry with the highest score is evicted. 
+
+#### 7.8.2.2. Near Cache Example for Map
+
+The following is an example configuration for a Near Cache defined in the `mostlyReadMap` map. According to this configuration, the entries are stored as `object`'s in this Near Cache and eviction starts when the count of entries reaches `5000`; entries are evicted based on the `lru` (Least Recently Used) policy. In addition, when an entry is updated or removed on the member side, it is eventually evicted on the client side.
+
+**Declarative:**
+
+```
+{
+    "nearCaches": [
+        {
+            "name": "mostlyReadMap",
+            "inMemoryFormat": "object",
+            "invalidateOnChange": true,
+            "evictionPolicy": "lru",
+            "evictionMaxSize": 5000,
+        }
+    ]
+}
+```
+
+**Programmatic:**
+
+```javascript
+var nearCacheConfig = new Config.NearCacheConfig();
+nearCacheConfig.name = "mostlyReadMap";
+nearCacheConfig.inMemoryFormat= InMemoryFormat.OBJECT;
+nearCacheConfig.invalidateOnChange = true;
+nearCacheConfig.evictionPolicy = EvictionPolicy.LRU;
+nearCacheConfig.evictionMaxSize = 5000;
+
+cfg.nearCacheConfigs['mostlyReadMap'] = nearCacheConfig;
+```
+
+#### 7.8.2.3. Near Cache Eviction
+
+In the scope of Near Cache, eviction means evicting (clearing) the entries selected according to the given `evictionPolicy` when the specified `evictionMaxSize` has been reached.
+
+The `evictionMaxSize` defines the entry count when the Near Cache is full and determines whether the eviction should be triggered. 
+
+Once the eviction is triggered the configured `evictionPolicy` determines which, if any, entries must be evicted.
+
+#### 7.8.2.4. Near Cache Expiration
+
+Expiration means the eviction of expired records. A record is expired:
+
+- if it is not touched (accessed/read) for `maxIdleSeconds`
+
+- `timeToLiveSeconds` passed since it is put to Near Cache
+
+The actual expiration is performed when a record is accessed: it is checked if the record is expired or not. If it is expired, it is evicted and `undefined` is returned as the value to the caller.
+
+
+#### 7.8.2.5. Near Cache Invalidation
+
+Invalidation is the process of removing an entry from the Near Cache when its value is updated or it is removed from the original data structure (to prevent stale reads). See the [Near Cache Invalidation section](https://docs.hazelcast.org/docs/latest/manual/html-single/#near-cache-invalidation) in the Hazelcast IMDG Reference Manual.
```

Comment:
the original data structure (to prevent stale reads). -> the original map (to prevent stale reads).